pynsee/metadata/get_legal_entity.py

```diff
@@ -50,7 +53,7 @@ def get_legal_entity(codes, print_err_msg=True, update=False):
 
         data_final = data_final.rename(columns={"intitule": "title"})
         data_final.to_pickle(file_legal_entity)
-        print(f"Data saved: {file_legal_entity}")
+        logger.debug(f"Data saved: {file_legal_entity}")

```

Comment:
set to info

Reply:
Should be consistent with previous one about file's storing.

Reply:
yes, if I missed some, please move all data related messages to info: these can be useful to anyone using pynsee to confirm that everything went well and are not necessarily used to debug the library